swap.sh

```diff
@@ -2,4 +2,70 @@
 set -e
 set -o pipefail
 
-echo "Initializing swap space"
\ No newline at end of file
+readonly SWAP_FILE_PATH="/home/shippable/.__sh_swap__"
```

Comment:
- change this to `/root/.__sh_swap__`
- add an env in global node variables called SHIPPABLE_SWAP_PATH which points to the file 
- use the env to refer to file